content/usgs-packages/Application_Intro.Rmd

```diff
@@ -32,14 +32,16 @@ set.seed(1)
 
 ## Lesson Summary
 
-TBD
+This lesson will revisit the packages taught in previous lessons and use some of their functions to create a common data analysis workflow in a modular and reproducible manner. Then, you can learn how a modular approach compares to traditional analyses.
```

Comment:
this is an intriguing framing for the lesson. i'm looking forward to seeing how the rest of it plays out. is it worth noting here that everybody uses some modularity all the time? we use Excel, Word, Google sites...all these things are somebody else's software tool that helps us do our own work. i guess the difference is that somebody might be tempted to create their own URLs for an API call to NWIS Web, for example, whereas they'd never think/bother to try to build their own version of Word. anyway, i'm thinking it might be hard to follow through on the promise in the second sentence above...see how it pans out, then revise this if needed.